Tests/IntegrationTests.Shared/InstanceTests.cs

```diff
@@ -62,6 +62,16 @@ public void GetInstanceWithJustFilenameTest()
             // Arrange, act and "assert" that no exception is thrown, using default location + unique name
             Realm.GetInstance(specialRealmName).Close();
         }
+        
+        [Test]
```

Comment:
Can we put the unit tests inside the PCL? I figured that opened a bigger can of worms plus I wanted to explicitly test calling functions **from** native code, that 

1. use a PCL object on a realm in the native app and
2. go through the whole Realm creation **in** the PCL